chainerx_build_helper.py

```diff
@@ -76,9 +83,10 @@ def build_extension(self, ext):
                 '-DCMAKE_LIBRARY_OUTPUT_DIRECTORY_{}={}'.format(
                     build_type.upper(), extdir)]
 
-            if sys.maxsize > 2**32:
-                cmake_args += ['-A', 'x64']
-            build_args += ['--', '/m']
+            if not self.use_ninja:
```

Comment:
Why is this fix needed?
I think it's favorable to write a comment.

Reply:
simply because ninja does not accept these args

Reply:
`cmake_args` is arguments for CMake. Is `-A` also incompatible with ninja?

Reply:
```
CMake Error at CMakeLists.txt:2 (project):
  Generator

    Ninja

  does not support platform specification, but platform

    x64

  was specified.
```
reconfirmed.

Reply:
Thank you for confirmation.
The situation is a bit complicated and I think it's not this PR's responsibility to make a comment here...